regen/opcode.pl

```diff
@@ -821,27 +830,12 @@ sub print_PL_op_private_tables {
     }
 
     print $fh <<EOF;
-START_EXTERN_C
-
-#ifndef DOINIT
-
-/* data about the flags in op_private */
-
-EXTCONST I16  PL_op_private_bitdef_ix[];
-EXTCONST U16  PL_op_private_bitdefs[];
-EXTCONST char PL_op_private_labels[];
-EXTCONST I16  PL_op_private_bitfields[];
-EXTCONST U8   PL_op_private_valid[];
-
-#else
-
-
 /* PL_op_private_labels[]: the short descriptions of private flags.
  * All labels are concatenated into a single char array
  * (separated by \\0's) for compactness.
  */
 
-EXTCONST char PL_op_private_labels[] = {
+const char * PL_op_private_labels = (const char []) {
 $PL_op_private_labels
 };
```

Comment:
Compound literal syntax is disallowed by perlhacktips for C++ compatibility.

This also changes PL_op_private_labels from const to non-const.

(it was a const array of char, this changes it to a non-const pointer to const char.)

Similarly below.

I don't understand why you made this change.

Reply:
It was part of discussion about my earlier commit (2 years ago) - to move variable initialization from `.h` files to `.c` files, get rid of `INIT` macro, and redefinition of various macros for `globals.c`